pkg/cmd/server/start/start_master.go

```diff
@@ -584,6 +584,8 @@ func StartAPI(oc *origin.MasterConfig, kubeAPIServerConfig *master.Config, infor
 		if err != nil {
 			return err
 		}
+		config.GenericConfig.AuditBackend = kubeAPIServerConfig.GenericConfig.AuditBackend
```

Comment:
This was a good usage of your `GetAuditConfig` bit.  The `kubeapiserverconfig` goes away here: https://github.com/openshift/origin/pull/16582

Reply:
sigh... I'll revert it back